### DISCUSSION TITLE
How to transition/animate conditional styles

### DISCUSSION THREAD

**Discussion by Maxson52** [nandorojo/dripsy#282] [Q&A] [answered]
How can I transition between conditional styles similar to how CSS transitions work?

```tsx
// Example: animate when `foo` changes
<Text sx={foo ? {color: "blue"} : {color: "orange"}}></Text>
```

My understanding is that I could use the animation API in React Native, however it seems a bit boilerplatey. Is this done easily using Moti, Legend Motion, or similar? Can it be done natively using Dripsy?

**Comment by nandorojo** [comment, marked as answer]
use Moti! It’s not handled by Dripsy, I purposefully made Moti a separate library.

**Reply by Maxson52** [reply]
Cool, will check it out!
